packages/appsync-emulator-serverless/__test__/schemaTest.test.js

```diff
@@ -314,4 +314,39 @@ describe('creates executable schema', () => {
 
     expect(result).toMatchObject({ data: { jsonTest: { test: 'yup' } } });
   });
+
+  it('custom scalar support', async () => {
+    async function asyncForEach(array, callback) {
+      for (let index = 0; index < array.length; index += 1) {
+        await callback(array[index], index, array);
```

Comment:
Do we need to pass the second and third arguments here?

Reply:
Probably not. I just recycled an async foreach I had handy.